template/data/fr/global.json

```
@@ -0,0 +1,116 @@
+{
+  "globalFR": {
+    "id": 2,
+    "locale": "fr",
+    "metaTitleSuffix": "Strapi Corporate",
+    "metadata": {
+      "metaTitle": "Strapi starter for Corporate Sites",
+      "metaDescription": "Créer votre site corporate avec Strapi",
+      "twitterCardType": "summary",
+      "twitterUsername": "strapijs",
+      "shareImage": null
+    },
+    "notificationBanner": {
+      "text": "Cette page a été crée avec le Strapi starter pour les sites Corporate.\n [Voir les autres Strapi starters](https://strapi.io/starters)",
+      "type": "info"
+      
+    },
+    "navbar": {
+      "links": [
+        {
+          "url": "/tarifs",
+          "newTab": false,
+          "text": "Tarifs"
+        },
+        {
+          "url": "/nous-contacter",
+          "newTab": false,
+          "text": "Nous Contacter"
+        }
+      ],
+      "button": {
+        "url": "#",
+        "newTab": false,
+        "text": "S'Inscrire",
+        "type": "secondary"
+      },
+      "logo" : {
+        "image": null,
+        "url": "fr"
+      }
+    },
+    "footer": {
+      "smallText": "© Copyright My Company™",
+      "columns": [
+        {
+          "title": "Produit",
+          "links": [
+            {
+              "url": "#",
+              "newTab": false,
+              "text": "Features"
+            },
+            {
+              "url": "#",
+              "newTab": false,
+              "text": "S'Inscrire"
+            }
+          ]
+        },
+        {
+          "title": "Legal",
+          "links": [
+            {
+              "url": "#",
+              "newTab": false,
+              "text": "Confidentialité"
+            },
+            {
+              "url": "#",
+              "newTab": false,
+              "text": "Mentions légales"
+            }
+          ]
+        },
+        {
+          "title": "Company",
```

```suggestion
          "title": "Entreprise",
```

(sounds a bit weird but I couldn't find a better alternative)